tests/unit/src/folio-runner.spec.js

```diff
@@ -1,14 +1,15 @@
 jest.mock('child_process');
 jest.mock('saucelabs');
 jest.mock('fs');
+jest.mock('sauce-testrunner-utils');
 jest.mock('../../../src/reporter');
 const path = require('path');
 const { run } = require('../../../src/folio-runner');
 const childProcess = require('child_process');
 const { EventEmitter } = require('events');
 const SauceLabs = require('saucelabs').default;
 const fs = require('fs');
-const utils = require('../../../src/utils');
+const remoteUtils = require('sauce-testrunner-utils');
```

Comment:
what do you think about renaming this to be more "sauce-runner" specific `runnerUtils`, `sttUtils`?